ts/screens/wallet/WalletHomeScreen.tsx

```diff
@@ -519,10 +519,9 @@ class WalletHomeScreen extends React.PureComponent<Props, State> {
             anyHistoryPayments || anyCreditCardAttempts
           );
 
-    const footerContent =
-      pot.isSome(potWallets) && !this.newMethodAdded
-        ? this.footerButton(potWallets)
-        : undefined;
+    const footerContent = pot.isSome(potWallets)
+      ? this.footerButton(potWallets)
```

Comment:
May it be a good moment to remove this navigation param? It is now only used in another point of the component and it may be useless there too...at least we may reduce another tricky part of this hydra component 😄

Reply:
Yeah! we talked about it with @fabriziofff 
It will checked in a dedicated story/PR